corehq/pillows/mappings/app_mapping.py

```diff
@@ -70,7 +73,11 @@
                                                                                   'type': 'object'},
                                                                         'show': {'type': 'boolean'}},
                                                          'type': 'object'},
-                                           'case_type': {'type': 'string'},
+                                           'case_type': {'fields': {'case_type': {'index': 'analyzed',
+                                                                                  'type': 'string'},
+                                                                    'exact': {'index': 'not_analyzed',
+                                                                              'type': 'string'}},
+                                                         'type': 'multi_field'},
```

Comment:
I mentioned it in my commit message for this change, but basically I'm making a "not_analyzed" version of this field so the faceted results will be case sensitive, like it currently is with couch.